homeassistant/components/velux/cover.py

```diff
@@ -2,14 +2,7 @@
 from pyvlx import OpeningDevice, Position
 from pyvlx.opening_device import Awning, Blind, GarageDoor, Gate, RollerShutter, Window
 
-from homeassistant.components.cover import (
-    ATTR_POSITION,
-    SUPPORT_CLOSE,
-    SUPPORT_OPEN,
-    SUPPORT_SET_POSITION,
-    SUPPORT_STOP,
-    CoverEntity,
-)
+from homeassistant.components import cover
```

Comment:
We prefer to import only the names we need instead of the whole module. It's just a style thing.

Reply:
Done :-)